cernopendata/modules/fixtures/data/records/atlas-ATL-PHYS-PUB-2022-039.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "abstract": {
+      "description": "<p>FIXME This dataset serves for performance studies of constituent-based jet taggers on large radius boosted top quark jets reconstructed from optimized jet input objects in simulated collisions at √ 𝑠 = 13 TeV. Several taggers which consider all the information contained in the kinematic information of the jet constituents are tested, and compared to a tagger which relies on high-level summary quantities similar to the taggers used by ATLAS in Runs 1 and 2. Several constituent based taggers are found to out-perform the high level quantity based tagger, with the best achieving a factor of two increase in background rejection across the kinematic range.</p> <p>FIXME This dataset consists of two files, the training data and the test data, both available in H5 format. The dataset semantics and code usage examples are linked below.</p>"
```

Comment:
This is a draft PR to finalise text.  Current looks:

![draft](https://user-images.githubusercontent.com/517546/187487537-de810aa0-ca02-4f41-8e8b-8df25722781d.png)